File: print_err.py
import sys

with open("./autumn-harvest/src/store.rs", "r") as f:
    content = f.read()

search_str = """        assert!(result.unwrap_err().to_string().contains("event ID overflow"));"""

replace_str = """        println!("err={}", result.unwrap_err().to_string());
        assert!(false);"""

content = content.replace(search_str, replace_str)

with open("./autumn-harvest/src/store.rs", "w") as f:
    f.write(content)

File: print_err.rs
use autumn_harvest::store;
use autumn_harvest::types::{ActivityExecId, ExecutionId};
use autumn_harvest::event::WorkflowEvent;
use std::convert::TryFrom;
fn main() {
    let exec_id = ExecutionId::new();
    let events = vec![
        WorkflowEvent::ActivityCompleted {
            activity_id: ActivityExecId::new(),
            output: serde_json::json!("ok"),
        },
        WorkflowEvent::WorkflowCompleted {
            output: serde_json::json!(null),
        },
    ];

    let result = store::events_to_insert_rows_from(exec_id, &events, i32::MAX);
    println!("Err: {:?}", result.unwrap_err().to_string());
}
